client/src/modules/distribution_center/modals/distribution_key.modal.js

```diff
@@ -0,0 +1,77 @@
+angular.module('bhima.controllers')
+  .controller('DistributionKeyModalController', DistributionKeyModalController);
+
+DistributionKeyModalController.$inject = [
+  '$state', 'NotifyService', 'DistributionCenterService', 'PayrollConfigurationService',
+  'ExchangeRateService', 'SessionService', '$uibModalInstance', 'PatientInvoiceService',
+  'FeeCenterService', 'util', 'settings', '$translate',
+];
```

Comment:
Could you remove : PayrollConfigurationService, ExchangeRateService, PatientInvoiceService, and util since they are not used